dynamiqs/mesolve/mesolve.py

```diff
@@ -160,6 +160,36 @@ def mesolve(
             'Argument `jump_ops` must be a non-empty list, otherwise consider using'
             ' `sesolve`.'
         )
+
+    # check all jump ops are batched in a similar way or not at all
```

Comment:
This whole piece of code should probably not be inside `mesolve`, but inside `batch_jump_ops` to have all batching taken care of in the same place.

Reply:
It's not easy to put evything in `batch_jump_ops` because it would then have to also include `to_tensor` inside of it and make a function with a lot of arguments and much bigger than what we do for H and rho. Instead, I made another function `prepare_jump_ops_batching` that does the pre-formatting